go.mod

```diff
@@ -5,20 +5,17 @@ go 1.12
 require (
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/golang/protobuf v1.3.5
-	github.com/gopherjs/gopherjs v0.0.0-20191106031601-ce3c9ade29de // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.3
 	github.com/sirupsen/logrus v1.4.2
-	github.com/smartystreets/assertions v1.0.1 // indirect
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/viper v1.6.2
-	github.com/stretchr/testify v1.4.0 // indirect
-	golang.org/x/crypto v0.0.0-20191122220453-ac88ee75c92c // indirect
+	github.com/zcash-hackworks/lightwalletd v0.3.0
```

Comment:
There shouldn't be any references to `zcash-hackworks` in this file (or anywhere here). If you run `make update-grpc` it should fix these. (This makefile target name is a bit misleading, because it updates more than just grpc stuff.)